@Test public void should_fail_list_asserts_on_non_list_objects_even_with_asList(){
  Object nonList=new Object();
  assertThatExceptionOfType(AssertionError.class).isThrownBy(() -> assertThat(nonList).asList().isSorted()).withMessageContaining(format("an instance of:%n  <java.util.List>%nbut was instance of:%n  <java.lang.Object>"));
}
